website/docs/r/frontdoor.html.markdown

```diff
@@ -167,7 +167,7 @@ The `backend_pool_health_probe` block supports the following:
 
 * `path` - (Optional) The path to use for the Health Probe. Default is `/`.
 
-* `protocol` - (Optional) Protocol scheme to use for the Health Probe. Defaults to `Http`.
+* `protocol` - (Optional) Protocol scheme to use for the Health Probe. Defaults to `Http`. Possible values are `Http` and `Https`.
```

Comment:
Can we switch the order around for the possible values and the default value

```suggestion
* `protocol` - (Optional) Protocol scheme to use for the Health Probe. Possible values are `Http` and `Https`. Defaults to `Http`.
```

Reply:
updated